pkg/hintrunner/zero/zerohint.go

```diff
@@ -107,7 +107,7 @@ func GetHintFromCode(program *zero.ZeroProgram, rawHint zero.Hint, hintPC uint64
 		return createUint256SqrtHinter(resolver)
 	case uint256MulDivModCode:
 		return createUint256MulDivModHinter(resolver)
-	// Signature hints
+		// Signature hints
```

Comment:
i think they are all shifted and only the ` // Match hints ` (first one) is in the right place

Reply:
yeah it shouldnt be. At some point, some of the comment for hints were placed like this. We aligned all of them correctly in a previous PR.

Reply:
My bad this is a mistake. I'll push a fix.